webapp/package.json

```diff
@@ -26,7 +26,7 @@
     "eslint": "4.10.0",
     "eslint-config-prettier": "^2.6.0",
     "eslint-plugin-prettier": "^2.3.1",
-    "eslint-plugin-react": "^7.5.1",
+    "eslint-plugin-react": "7.4.0",
```

Comment:
Great! Maybe not for this PR, but what do you guys think about pinning up dependencies version and using a tool like greenkeeper.io so we don't have new uncontrolled errors and we can control version upgrades? @abarmat @cazala @NicoSantangelo